designs/UI/myJS.js

```diff
@@ -4,14 +4,41 @@ function validateForm()
     var b=document.forms["regForm"]["email"].value;
     var c=document.forms["regForm"]["password"].value;
     var d=document.forms["regForm"]["verpassword"].value;
-    if (a==null || a=="",b==null || b=="",c==null || c=="",d==null || d=="")
+
```

Comment:
Why is your file name myJS.js? Have an appropriate name for the file

Reply:
should I rename it to yummy-recipes.js?